app/helpers/axios.js

```diff
@@ -0,0 +1,12 @@
+const axios = require('axios');
+
+const logger = require('../logger');
+const { badGatewayError } = require('../errors');
+
+exports.getWeetRandom = (method, url, data = '') =>
+  axios({ method, url, data })
+    .then(response => response.data.joke)
```

Comment:
And here only return `response.data` in accordance with de above comment.

Reply:
it's was an oversight. Sorry!